Frontend/rasandbox/src/App.vue

```diff
@@ -1,70 +1,40 @@
 <template>
-  <div :class="themeClass">
-    <user-modal
-      v-if="showModal"
-      :class="{ 'is-active': showModal }"
-      :favGameList="favGameList"
-      @close="showModal = false"
-    />
+  <div class="betterSteam" :class="themeClass">
+    <user-modal v-if="showModal" class="is-active" @close="showModal = false" />
     <filter-modal
       v-if="showFilter"
-      :class="{ 'is-active': showFilter }"
+      class="is-active"
```

Comment:
das ist eine Klasse die in die `<Modal>` Komponente gehört und statisch dran hängt. Das sollte deine konsumierende Komponente (App.vue) gar nicht wissen.